Sources/AWSSDKSwiftCore/Exports.swift

```diff
@@ -0,0 +1,2 @@
+@_exported import AWSSignerV4
+@_exported import NIO
```

Comment:
I'm not sure if we really want to expose NIO?

Reply:
Pretty anytime someone is interacting with AWSSDKSwift they are required to deal with EventLoopFutures which requires them to `import NIO`. We could expose selected objects like EventLoopFuture, EventLoopGroup, ByteBuffer maybe

Reply:
See Changes, just exporting the objects we expose in our public APIs